### PR TITLE
🐕 Added place.dog to animals section

### PR DESCRIPTION
**Pull request by joshghent**
Thanks for the awesome project! Hope you enjoy the doggos 🐶 

- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by dsachin — APPROVED** [review]
LGTM

**Comment by marekdano** [comment]
Thanks for the contribution @joshghent!

Can you please resolve the conflict?

**Comment by joshghent** [comment]
> Thanks for the contribution @joshghent!
> 
> Can you please resolve the conflict?

Done!

File: README.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ API | Description | Auth | HTTPS | CORS |
 | [HTTPCat](https://http.cat/) | Cat for every HTTP Status | No | Yes | Unknown |
 | [IUCN](http://apiv3.iucnredlist.org/api/v3/docs) | IUCN Red List of Threatened Species | `apiKey` | No | Unknown |
 | [Movebank](https://github.com/movebank/movebank-api-doc) | Movement and Migration data of animals | No | Yes | Unknown |
+| [PlaceDog](https://place.dog) | Placeholder Dog pictures | No | Yes | Yes |
+| [PlaceGOAT](https://placegoat.com/) | Placeholder goat images | No | Yes | Unknown |
```
Comment (marekdano) [inline README.md:73]: `https://placegoat.com/` doesn't seem to work. Can you please update the URL or remove the PlaceGOAT record?

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for resolving the conflict!
Can you please look at my comment?

File: README.md (hunk 1 above)
Comment (joshghent) [reply README.md]: Yeah sure thing!

**Review by marekdano — APPROVED** [review]

**Pull request merged**